.circleci/config.yml

```diff
@@ -405,20 +396,10 @@ workflows:
           matrix:
             parameters:
               ansible_version: ["2_10", "3_4", "4_10"]
-              agent_version: ["5", "7"]
+              agent_version: ["7"]
```

Comment:
Just curious, do you happen to know why we are only testing agent 7 here and not also agent 6?

Reply:
im not sure why, the initial PR that added this test only ran it on agent 5 and 7. But i added testing on Agent 6 and it works as expected